src/app/ngrx/reducers/episode-metrics.reducer.ts

```diff
@@ -5,12 +5,10 @@ export interface EpisodeMetricsModel {
   id: number;
   guid?: string;
   page?: number;
-  charted?: boolean;
```

Comment:
So we're not tracking the "charted" state anymore?

Reply:
It's only on the router state now and is using selector composition to combine it with metrics for the chart data selector